### ISSUE TITLE
`unable to find profile` errors when a config name (not path) is used

### ISSUE THREAD

**Issue by omajid** [rpminspect/rpminspect#940]
When I use a path with `--config` flag, `--profile` works. When I use a name with `--config`, `--profile` gives me errors about not finding a profile:

```
$ rpm -q rpminspect rpminspect-data-redhat rpminspect-data-fedora
rpminspect-1.11-0.1.202210101546gitb89132b.fc36.x86_64
rpminspect-data-redhat-1.7-0.1.202209291832gita40e191.fc36.noarch
rpminspect-data-fedora-1.10-0.1.202208011434git0eb44e0.fc36.noarch

# I think I have all the files
$ ls -alh  /usr/share/rpminspect/redhat.yaml  /usr/share/rpminspect/profiles/redhat/scl.yaml
-rw-r--r--. 1 root root 146 Sep 29 14:30 /usr/share/rpminspect/profiles/redhat/scl.yaml
-rw-r--r--. 1 root root 32K Sep 29 14:30 /usr/share/rpminspect/redhat.yaml

# full path used with --config, --profile works
$ rpminspect -vvv --config /usr/share/rpminspect/redhat.yaml --profile scl rh-dotnet60-aspnetcore-runtime-6.0-6.0.10-1.el7_9.x86_64.rpm
Running license inspection...        pass
^C

# vendor name used with config, --profile fails
$ rpminspect -vvv --config redhat --profile scl rh-dotnet60-aspnetcore-runtime-6.0-6.0.10-1.el7_9.x86_64.rpm
rpminspect: *** unable to find profile 'scl'
```

In either case, passing a full path to `--profile` doesn't work.

**Comment by dcantrell** [comment]
Well that's no good.  I can fix that.

**Comment by dcantrell** [comment]
OK, I should have read this more.  The -c option requires the full path to the base configuration file that rpminspect should use.  This is why the data packages provide wrapper scripts to invoke rpminspect with that config file, like /usr/bin/rpminspect-redhat

The -p option requires the profile's file basename without the .yaml extension.  So specifying "scl" is correct.  The behavior you are seeing is correct.

I can modify things so the profile can be specified with a full path.  The profile directory is set by the main configuration file which is why I set it up that way.

Is there a reason you want to specify the full path to the profile and not just "scl"?

**Comment by omajid** [comment]
Hey, so I am re-reading the man pages and I can see that I misunderstood the requirement that `--config` needs a full path and `--profile` needs a specific name (not a file, nor a path). The difference seems a bit odd, but not a big deal.

I think I was mostly thrown off by the error message misleading me into thinking it's an issue with `--profile` when I have just used `--config` incorrectly:

```
$ rpminspect -vvv --config redhat --profile scl rh-dotnet60-aspnetcore-runtime-6.0-6.0.10-1.el7_9.x86_64.rpm
rpminspect: *** unable to find profile 'scl'
```

If it said that the `--config` flag is incorrect, I would have figured out my mistake earlier. Maybe something like this?

```
$ rpminspect -vvv --config redhat --profile scl rh-dotnet60-aspnetcore-runtime-6.0-6.0.10-1.el7_9.x86_64.rpm
rpminspect: *** unable to find config $(pwd)/redhat
```

Where `$(pwd)` is replaced by the actual path so I can tell that I used the `--config` option wrong.
